SDL/Module.cu

```diff
@@ -526,23 +532,21 @@ void SDL::loadModulesFromFile(struct modules& modulesInGPU, uint16_t& nModules,
     cudaMemcpyAsync(modulesInGPU.partnerModuleIndices, host_partnerModuleIndices, sizeof(uint16_t) * nModules, cudaMemcpyHostToDevice, stream);
     cudaStreamSynchronize(stream);
 
-    cudaFreeHost(host_detIds);
-    cudaFreeHost(host_layers);
-    cudaFreeHost(host_rings);
-    cudaFreeHost(host_rods);
-    cudaFreeHost(host_modules);
-    cudaFreeHost(host_subdets);
-    cudaFreeHost(host_sides);
-    cudaFreeHost(host_eta);
```

Comment:
What happened to the host_eta and host_r frees here? I think these are the missing 2

Reply:
Thank you for pointing this out, just added the fix.